x/distribution/model.go

```diff
@@ -146,3 +152,22 @@ func (b *RevenueBucket) GetRevenue(db weave.KVStore, revenueID []byte) (*Revenue
 	}
 	return rev, nil
 }
+
+// Create adds given revenue instance to the store and returns the ID of the
+// newly inserted entity.
+func (b *RevenueBucket) Create(db weave.KVStore, rev *Revenue) (orm.Object, error) {
+	key, err := b.idSeq.NextVal(db)
+	if err != nil {
+		return nil, errors.Wrap(err, "cannot acquire revenue ID")
+	}
+	if a, err := RevenueAccount(key); err != nil {
```

Comment:
Same as in `aswap` - I think we have to make it work either one way or another in all the extensions we upgrade.